### PR TITLE
Data 2976 adventurer s armory equipment tricks

### PR DESCRIPTION
**Pull request by evilpixie87**
Activation of equipment tricks in Adventurer's Armory

### CODE REVIEW

File: data/pathfinder/paizo/player_companion/adventurers_armory/aa_feats.lst [hunk 1]
```diff
@@ -9,7 +9,7 @@ SOURCELONG:Adventurer's Armory	SOURCESHORT:AA	SOURCEWEB:http://paizo.com/product
 # Ability Name			Category of Ability	Type			Required Ability					Description																																																																							Source, Long Desc.			Source Page		Benefits
 Rapid Reload (Spear-thrower)	CATEGORY:FEAT		TYPE:Combat									DESC:You can reload your spear-thrower quickly.																																																																							SOURCEPAGE:p.8	BENEFIT:You can reload a spear-thrower as a free action.
 
-#TODO Equipment Trick p.22
+Equipment Trick				CATEGORY:FEAT				TYPE:Combat						PRETOTALAB:1																																					PRETEXT:Base attack bonus +1.																			DESC:You understand how to use a certain item in combat.							SOURCEPAGE:p.22			BENEFIT:You may use any equipment tricks relating to the item if you meet the appropriate trick requirements. If the item would normally be considered an improvised weapon, you may treat it as a normal weapon or an improvised weapon, whichever is more beneficial for you. You can gain the Equipment Trick feat multiple times. Each time you take the feat, it applies to a new type of equipment.		STACK:NO	MULT:YES	CHOOSE:ABILITY|Special Ability|TYPE=EquipmentTrick
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/adventurers_armory/aa_feats.lst]: I see a selection, but is there any point to the selection?

File: data/pathfinder/paizo/player_companion/adventurers_armory/aa_abilities.lst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# CVS $Revision: 12957 $ $Author: jbaril $ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
+SOURCELONG:Adventurer's Armory	SOURCESHORT:AA	SOURCEWEB:http://paizo.com/products/btpy8dmf?Pathfinder-Player-Companion-Adventurers-Armory	SOURCEDATE:2010-04
+# Original Entry by: Stefan Radermacher
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/adventurers_armory/aa_abilities.lst]: If @Zaister made this, why are you submitting it???

File: data/pathfinder/paizo/player_companion/adventurers_armory/aa_feats.lst (hunk 1 above)
Comment (evilpixie87) [inline data/pathfinder/paizo/player_companion/adventurers_armory/aa_feats.lst]: Yes. There are several different pieces of equipment, so the player must pick one for the trick.

File: data/pathfinder/paizo/player_companion/adventurers_armory/aa_abilities.lst (hunk 2 above)
Comment (evilpixie87) [inline data/pathfinder/paizo/player_companion/adventurers_armory/aa_abilities.lst]: He did most of this source. All I did was get the one piece he didn't do... I CAN add my name, but honestly, I wasn't worried about it.

**Pull request merged**